src/evaluation/unit tests/test_CharHitAtK.py

```diff
@@ -0,0 +1,60 @@
+from src.classes.model_result import ModelResult
+from src.classes.prediction import Prediction
+from src.classes.text_part import TextPart
+from src.evaluation.Classes.CharHitAtK import CharHitAtK
+from src.model.stub_model import StubModel
+
```

Comment:
@itaiassraf  , i have started it  to help you a little . please add another use cases that you may think about